### DISCUSSION TITLE
关于耗电，和占用

### DISCUSSION THREAD

**Discussion by xzayp1324** [gkd-kit/gkd#707] [Announcements]
### 一些验证

- [X] 1.GKD 默认不提供任何规则, 你可以查看 [GKD 官方文档](https://gkd.li) 后自行编写规则或者导入远程订阅, 请不要再提出类似想要XXX规则这种问题
- [X] 2.GKD 默认不提供任何规则, 你可以查看 [GKD 官方文档](https://gkd.li) 后自行编写规则或者导入远程订阅, 请不要再提出类似想要XXX规则这种问题
- [X] 3.GKD 默认不提供任何规则, 你可以查看 [GKD 官方文档](https://gkd.li) 后自行编写规则或者导入远程订阅, 请不要再提出类似想要XXX规则这种问题
- [X] 1.请 **确保** [已有的问题](https://github.com/gkd-kit/gkd/issues?q=is%3Aissue) 中没有人提交过相似issue，否则请在已有的issue下进行讨论
- [X] 2.请 **确保** [已有的问题](https://github.com/gkd-kit/gkd/issues?q=is%3Aissue) 中没有人提交过相似issue，否则请在已有的issue下进行讨论
- [X] 3.请 **不要** 开启重复相关的 issue，这将导致别人搜索 issue 时出现无关的低质量信息, 否则你的问题将会被直接关闭甚至删除
- [X] 请 **务必** 给issue填写一个简洁明了的标题，以便他人快速检索
- [x] 请 **确保** 你想要的功能在 [releases](https://github.com/gkd-kit/gkd/releases/latest) 发布的最新版本(包含测试版本)上没有找到 (如果不是请先更新到最新版本验证后再提交问题)

### 新功能描述

看到有人说过耗电问题了，但还是想吐槽一下，占用和耗电都比李跳跳多，跳跳一天也才10毫安左右，具体原理我也不懂，可以再狠狠的优化一下吗，直接订阅规则这个确实比李跳跳方便`

![Screenshot_20240816_063314](https://github.com/user-attachments/assets/8e5c5e0e-b0da-4ca3-844f-69ea6a4901f6)
![Screenshot_20240816_112226](https://github.com/user-attachments/assets/461cc4a0-4d9f-45bf-b111-2c0c840dc431)

**Comment by xzayp1324** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by lisonge** [comment]
你开太多规则了，你开一堆规则又想要不耗电

是不是有点异想天开

**Comment by xzayp1324** [comment]
> 你开了太多规则了，你开了一堆规则又想要不耗电
> 
> 是不是有点异想天开

不是很懂跳跳为什么这么小，我也有倒入上万条规则

**Comment by lisonge** [comment]
因为你导入是全是一堆空的，无用的，重复的规则

这些规则全是营销号批量复制粘贴出来的

**Reply by lisonge** [reply]
你的那些 `上万` 的规则，基本上都是源于 https://github.com/Snoopy1866/LiTiaotiao-Custom-Rules/tree/e5a163d9ca1156f5c46fa93ba88800caa7083b90 的重复复制粘贴然后加一堆空规则而来

**Reply by lisonge** [reply]
此外 抖音 这类软件的规则很耗电，你可以考虑关闭此类相关规则

**Comment by aoguai** [comment]
不知道你怎么开那么久前台的，而且3小时消耗那么多电量你加了多少规则才会这样。
我前台悬浮窗+权限都给，一天才37mAh，我的手机电池是5500mAh。37mAh占5500mAh的电池容量的大约0.67%。
![1](https://github.com/user-attachments/assets/2cfdd171-ffd8-4748-a305-83d48ef88316)
![image](https://github.com/user-attachments/assets/a1d308b7-1575-4142-a853-fa956ccb5a26)

**Comment by xfljy** [comment]
或许的确是你规则开多了，也可能是像我的之前不知道啥原因抽风一样（ https://github.com/orgs/gkd-kit/discussions/660 这个重启就好了）

这个软件正常的耗电应该是很低的（这里是开了两个订阅和一些本地规则的耗电量）
![3.jpg](https://github.com/user-attachments/assets/826c1745-4616-4d0e-b123-12e41a04dae3)


![2.png](https://github.com/user-attachments/assets/008ba0b9-1762-42d9-844d-ba12087be59f)
